vendor/assets/javascripts/react_rails_img.js.jsx.erb

```diff
@@ -4,15 +4,10 @@ prefix = Rails.application.config.assets.prefix
 should_compute_assets_path = Rails.env.production? # for performance
 if should_compute_assets_path
   images_dir = Rails.root.join('app/assets/images/').to_s
-  assets.each_file do |file_path|
+  Dir["#{images_dir}/**/*"].each do |file_path|
```

Comment:
@itaitay I would like to accept your PR if you can update the codes as I posted.